eoverdracht/leveranciersspecificatie.md

```diff
@@ -271,19 +271,19 @@ TODO: uitwerking in RFC opnemen voor https://github.com/nuts-foundation/nuts-spe
 
 Zoals beschreven in [§3.4](#34-scheiding-proces-en-autorisatie) scheiden we het overdrachtsproces van de autorisatie voor gegevenstoegang. Dit betekent dat er naast de task die het proces beheert ook een mechanisme aanwezig moet zijn dat de toegang tot gegevens regelt.
 
-In [§3.2](#32-wetgeving) staat beschreven op basis van welke grondslagen gegevens verwerkt mogen worden. Een verwijzing is een geldige grondslag, een mogelijke verwijzing \(in het geval van een aanmeldbericht\) in beginsel niet. In FHIR termen kunnen we een grondslag vastleggen in een consent resource.
+In [§3.2](#32-wetgeving) staat beschreven op basis van welke grondslagen gegevens verwerkt mogen worden. Een verwijzing is een geldige grondslag, een mogelijke verwijzing \(in het geval van een aanmeldbericht\) in beginsel niet.
 
-De verschillende processtappen in de verpleegkundige overdracht vereisen een bepaalde toegang tot gegevens. Evenzo zullen verschillende toestanden van de task resource leiden tot verschillende consent resources:
+De verschillende processtappen in de verpleegkundige overdracht vereisen een bepaalde toegang tot gegevens. Evenzo zullen verschillende toestanden van de task resource leiden tot bepaalde toegang:
 
-* Requested    → consent resource voor toegang tot aanmeldbericht
-* Accepted    → consent resource voor toegang tot aanmeldbericht
-* Rejected    → consent resources ingetrokken
-* Cancelled    → consent resources ingetrokken
-* Completed    → consent resource voor toegang tot overdrachtsbericht
+* Requested    → toegang tot aanmeldbericht
+* Accepted    → toegang tot aanmeldbericht
+* Rejected    → alle toegang ingetrokken
+* Cancelled    → alle toegang ingetrokken
+* Completed    → toegang tot overdrachtsbericht
 
-Het concept van een grondslag verbindt welke ontvangende partij toegang krijgt tot welke gegevens bij welke bronhouder inzake welke patiënt. En hoewel er voor het aanmeldbericht dus geen valide grondslag is om persoonsgegevens te verwerken, is het voor de implementatie wel handig om beide berichten via eenzelfde mechanisme toegankelijk te maken.
+Het concept van een grondslag verbindt welke ontvangende partij toegang krijgt tot welke gegevens bij welke bronhouder inzake welke patiënt. En hoewel er voor het aanmeldbericht dus geen valide grondslag is om persoonsgegevens te verwerken, is het voor de implementatie wel handig om beide berichten via eenzelfde mechanisme toegankelijk te maken. De grondslag is onderdeel van een speciaal autorisatie record, deze is beschreven in [RFC014](https://nuts-foundation.gitbook.io/drafts/rfc/rfc014-service-authorization).
```

Comment:
RFC014 bestaat nog niet. Deze link misschien weghalen of vanuit dit PR verwijzen naar een issue.

Reply:
Is being worked on in parallel.